src/providers/EtherspotTransactionKitContextProvider.tsx

```diff
@@ -240,11 +268,40 @@ const EtherspotTransactionKitContextProvider = ({
     return result;
   };
 
+  const getTransactionHash = async (
+    userOpHash: string,
+    batchId: number
+  ): Promise<string | null> => {
```

Comment:
Suggestion: allow the `timeout` and `retryInternal` to be configurable in `getTransactionHash` as optional parameters just incase a developer wants to override this